### DISCUSSION TITLE
Canvas doesn't show image on released apk

### DISCUSSION THREAD

**Discussion by Pierre-YvesLUDA** [iddan/react-native-canvas#270] [Q&A]
Hello,

I'm facing a issue with Canvas component, when I use it in dev mode it shows the image as intended. However, when I release an apk and I use it in release mode it doesn't show any image whereas I'm sure that images are correctly imported.

Also, I have the next warning : 
`Require cycle: node_modules\react-native-canvas\dist\Canvas.js -> node_modules\react-native-canvas\dist\Image.js -> node_modules\react-native-canvas\dist\Canvas.js
Require cycles are allowed, but can result in uninitialized 
values. Consider refactoring to remove the need for a cycle. `
I don't know if it's related or not but maybe it can help you to figure out the issue.

Thanks in advance!

**Comment by lilian-delouvy** [comment]
Hello, I'm currently encountering this issue. Did you find a fix ?
